packages/repository/test/acceptance/repository.acceptance.ts

```diff
@@ -71,6 +71,36 @@ describe('Repository in Thinking in LoopBack', () => {
     expect(stored).to.containDeep({extra: 'additional data'});
   });
 
+  it('enables strict delete by default', async () => {
+    const p = await repo.create({slug: 'pencil'});
+    p.name = 'Red Pencil';
+    await repo.save(p);
```

Comment:
`repo.create` has already created a persisted instance. No need to call `repo.save`. Please remove L76 and L77.